src/components/Chips/Chips.tsx

```diff
@@ -66,13 +65,9 @@ interface ChipsProps extends VibeComponentProps {
    */
   ariaLabel?: string;
   /**
-   * Should element be focusable & clickable - for backward compatability
+   * If you are using onMouseDown or onClick but do not want your chip to be used as a button (in terms of appearance or aria attributes)
```

Comment:
Until now if you were using onClick or onMouseDown it would not work unless you used also the clickable/isClickable extra prop which was breaking change (this prop is new) and also create less convenient API  (because we want that the chips will contain the correct behavior as default)

Reply:
removing these will break if people are using it with typescript, you can do but in Version 2.0